services/self-service/src/main/java/com/epam/dlab/backendapi/domain/EndpointDTO.java

```diff
@@ -23,14 +23,27 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
+import org.hibernate.validator.constraints.URL;
+import javax.validation.constraints.Pattern;
 
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class EndpointDTO {
-
+	/*
+	  the URL_REGEXP_VALIDATION constant is a template for URL pattern,
+	  i.e for url verification like "https://localhost:8084/a/$-*^.oLeh;/"
```

Comment:
I think we should not give examples of invalid URLs. So that we will rid of the comment because of the name of the field(**URL_REGEXP_VALIDATION**) give us information about regexp.

Reply:
done

Reply:
done